### DISCUSSION TITLE
Edit entry: Offer insight into the values of collapsed fields

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2935] [Feature Requests]
Fields in publish layouts that are collapsed give no indication whether a user has submitted data to them.

There should be some indication that collapsed field isn't empty. As I see it there are two options: 

1. Collapsed fields could give some visibility to their saved values. I recognize this gets less straightforward when you take all field types into account. 

```
> Product Color (Blue)

> Product Photo (blue_product_photo.jpg)

> Flag as on sale? (Yes)

> Related products (3 relationships)
```

2. Fields that are normally collapsed expand when they have user-submitted data in them. I know this is also complex, as I'm not sure there's a way to tell the difference between user-submitted data and a field's default state.

**Comment by intoeetive** [comment]
While this would work well for shorter field, I see some potential UX issue if the field is multi-line or contains a lot of data. Then collapsing the field would not really save much space.

I do see this being useful though, but we need to make sure it's controlled with a separate setting.

We could use the data in the format how it's being displayed in Entry Manager columns here
